pkg/operator/ceph/config/conditions.go

```diff
@@ -128,6 +137,61 @@ func ConditionInitialize(context *clusterd.Context, namespace, name string) {
 		Reason:  "",
 		Message: "",
 	})
+	setCondition(context, namespace, name, cephv1.Condition{
+		Type:    cephv1.ConditionAvailable,
+		Status:  v1.ConditionFalse,
+		Reason:  "",
+		Message: "",
+	})
+}
+
+// GetOSDDeployments returns an array containing the names of all OSD
+func GetOSDDeployments(context *clusterd.Context, namespace, AppName string) ([]string, error) {
+	var osdIDArray []string
+	listOpts := metav1.ListOptions{LabelSelector: fmt.Sprintf("app=%s", AppName)}
+	deployments, err := context.Clientset.AppsV1().Deployments(namespace).List(listOpts)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to get exsting deployment %q", AppName)
```

Comment:
wrong error message and typo "exsting"

Reply:
How about `return nil, errors.Wrapf(err, "failed to fetch osd deployment %q", AppName)`?